app/sensors/DropdownList.js

```diff
@@ -376,7 +372,7 @@ export default class DropdownList extends Component {
 							<Select
 								options={this.state.items}
 								clearable={false}
-								value={this.state.value}
+								value={_.isArray(this.state.value) ? this.state.value.join() : this.state.value}
```

Comment:
The initial state is a string for `value`, since we're using it as a string for SingleDropdownList. So, even for `MultiDropdownList`, it remains a string for the first render here. Can use `Array.isArray()` instead of lodash